changelogs/2.11.7.md

```diff
@@ -0,0 +1,111 @@
+# 2.11.7
+
+Date: 2025-05-28
+Tag: 2.11.7
+
+## Overview
+
+2.x is the old stable release series. Users are encouraged to update to the
+latest 3.x release.
+
+This is a bugfix release. It resolves 28 issues since the previous
+version.
+
+Please, consider the full list of user-visible changes below.
+
+## Compatibility
+
+Tarantool 2.x and 3.x are compatible in the binary data layout, client-server
+protocol, and replication protocol. It means that the updating may be performed
+with zero downtime for read requests and the-order-of-network-lag downtime for
+write requests.
+
+Please, follow the [upgrade procedure][upgrade] to plan your update actions.
+
+[upgrade]: https://www.tarantool.io/en/doc/latest/book/admin/upgrades/
+
+## Bugs fixed
+
+### Core
+
+* Fixed decoding of escape sequences for single-byte character codes from YAML.
+  Before the fix, single-byte character codes between `0x80` and `0xff` would
+  be erroneously converted to two-byte UTF-8 code points, for example, `\x80`
+  would be decoded as `\uC280` (gh-8782).
+* Fixed a bug where passwords are leaked into logs
+  when using `listen` and `bootstrap_leader` options (gh-10653).
+* Fixed a bug when cancelling a fiber waiting in WAL queue corrupted the
+  WAL (gh-11078).
+* Fixed a bug when the timestamps of snapshots created before the server restart
+  were not taken into account with `checkpoint_interval` enabled (gh-9820).
+* Fixed a bug when the sizes of `*.xlog` files created before the server restart
+  were not taken into account during the `checkpoint_wal_threshold` exceedance
+  checks (gh-9811).
+* Fixed a bug when transactions in the WAL queue were not rolled back on WAL
+  error (gh-11179).
+* Fixed bug when WAL queue was not flushed properly. In particular
+  when building an index of a vinyl space. In the latter case it could lead to
+  the new index missing data from transactions in the queue (gh-11118, gh-11119).
```

Comment:
```suggestion
* Fixed a bug when WAL queue was not flushed properly. In particular
  when building an index of a vinyl space. In the latter case it could lead to
  the new index missing data from transactions in the queue (gh-11118, gh-11119).
```